SPECS/kured/kured.spec

```diff
@@ -24,8 +24,8 @@
 %global debug_package %{nil}
 Summary:        Kubernetes daemonset to perform safe automatic node reboots
 Name:           kured
-Version:        1.9.1
-Release:        15%{?dist}
+Version:        1.15.0
```

Comment:
Can you please fill the PR checklist and also fill the testing method, local build/pipeline build.

Reply:
Updated, here is the buddy build! [[Link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=499195&view=results)]

Reply:
> Can you please fill the PR checklist and also fill the testing method, local build/pipeline build.

Thank you @mfrw , your feedback has been incorporated! Could I get your follow-up review when you have the chance? Thanks!